tests/torch/test_models/synthetic.py

```diff
@@ -259,14 +259,13 @@ def __init__(self):
         super().__init__()
         self.conv1 = create_conv(1, 2, 2, 2)
         self.conv2 = create_conv(2, 2, 2, 2)
+        self.relu6 = torch.nn.ReLU6()
 
-    @staticmethod
-    def _hswish(x: torch.Tensor) -> torch.Tensor:
-        return x * torch.nn.functional.relu6(x + 3) / 6
+    def _hswish(self, x: torch.Tensor) -> torch.Tensor:
+        return x * self.relu6(x + 3) / 6
```

Comment:
Use `ReLU6` module instead of `torch.nn.functional.relu6` to avoid issue with export in older torch version.